### DISCUSSION TITLE
Web UI for deleting an item from the article list view

### DISCUSSION THREAD

**Discussion by jman-schief** [wallabag/wallabag#7847] [Q&A]
When I want to delete an item, unless if I am wrong, the only way to accomplish this is to reach the "trash" icon at the far right of the entry
<details><summary>Screenshot</summary>

![screenshot-20241122-160220](https://github.com/user-attachments/assets/2f76f545-5439-41c4-86b4-0dc09696955b)

</details> 

On a 16:9 screen this is a bit impractical, to the point that I prefer to enable the multiselection, even when deleting one item
<details><summary>Screenshot</summary>

![screenshot-20241122-160237](https://github.com/user-attachments/assets/e622ddd9-2649-4b03-8e63-8a2f970ddbd2)

</details> 

I wonder if there is a discussion somewhere about this, or maybe someone has suggested an alternative UI?

Thanks

**Comment by nicosomb** [comment]
Oh there is a bug in the translation. Did you install wallabag or do you use app.wallabag.it? 

You can also delete an article thanks to your keyboard. You can read the howto page, the "use shortcuts" tab.

**Comment by jman-schief** [comment]
I am using app.wallabag.it but don't worry about the translations :)

Thanks for the tip about using the key `Del` to delete an article. This works fine if I am visiting an article. My question was more how to accomplish this from the list of articles.
